src/@packages/json-schema-to-jaam-schema/index.ts

```diff
@@ -0,0 +1,40 @@
+import { JaamSchema, JsonSchema } from "./types";
+
+export default function jsonSchemaToJaamSchema(schema: JsonSchema): JaamSchema {
+  return Object.entries(schema.properties).reduce(
+    (schema, [propName, options]) => {
+      const { type, description, format } = options;
+      const formattedType = (() => {
+        if (type === "string" && description) {
+          return description;
+        }
+
+        if (type === "string" && format) {
+          console.log(format);
+
+          if (format === "url") {
+            return "link";
+          }
+
+          return format;
+        }
+
+        return type;
+      })();
+
+      schema.properties[propName] = {
+        ...options,
+        type: formattedType,
+      };
+
+      return schema;
```

Comment:
```suggestion
      return {
        ...schema,
        properties: {
           ...schema.properties,
           [propName]: {
             ...options,
             type: formattedType,
           },
      };
```

reduce 내부 callback에서는 불변성이 유지되면 좋을 것 같아요!